src/server/services/wms/qgswmsparameters.cpp

```diff
@@ -2011,4 +2011,26 @@ namespace QgsWms
 
     return options;
   }
+
+  QMap<QString, QString> QgsWmsParameters::dimensionValues() const
+  {
+    QMap<QString, QString> dimValues;
+    const QStringList reservedNames { "TIME", "ELEVATION" };
```

Comment:
If an enum is defined in `QgsVectorLayerServerProperties` to specify dimensions, it won't be necessary to redefine these.